elements/lisk-validator/src/lisk_validator.ts

```diff
@@ -31,12 +31,14 @@ class LiskValidator {
 		for (const formatName of Object.keys(formats)) {
 			this.validator.addFormat(
 				formatName,
+				// eslint-disable-next-line import/namespace
```

Comment:
isn't this rules for imports only?

Reply:
I think this is showing error because of `as keyof typeof` for typescript

Reply:
If possible it would be nice to disable that one then. Approved anyway.

Reply:
I think this is the only case so far, so i would like to keep it for now.
If it continues to report wrong value, we can disable them